pilot/pkg/serviceregistry/kube/controller/controller.go

```diff
@@ -210,9 +210,9 @@ type Controller struct {
 
 	ambientIndex
 
-	// initialSyncTimedout is set to true after performing an initial processing timed out.
-	initialSyncTimedout *atomic.Bool
-	meshWatcher         mesh.Watcher
+	// initialSyncTimeout is set to true after performing an initial processing timed out.
+	initialSyncTimeout *atomic.Bool
```

Comment:
revert, this is to indicate it is a bool

Reply:
I didn't realize it had this meaning at all, how did you know?